composer.json

```diff
@@ -34,7 +34,9 @@
     "twig/inky-extra": "^3.3",
     "twig/cssinliner-extra": "^3.3",
     "twig/extra-bundle": "^3.3",
-    "symfony/ux-autocomplete": "^2.8"
+    "symfony/ux-autocomplete": "^2.8",
+    "doctrine/doctrine-bundle": "^2.11",
+    "doctrine/orm": "^3.0"
```

Comment:
Er waren geen off the shelf oplossingen? Had Tijs er niet 1 opgelijst in AC?

Reply:
De off the shelf oplossingen schreven weg naar een database of externe service, verder hebben we dan niet zoveel controle over de gelogde data